openreader/umbrel-app.yml

```
@@ -0,0 +1,37 @@
+manifestVersion: 1
+id: denny-openreader
+name: OpenReader WebUI
```

Probably better to just call it OpenReader as the WebUI part would anyways be cut on the home screen.
```suggestion
name: OpenReader
```

---

Good point!